frontend/ui/widget/inputdialog.lua

```diff
@@ -130,6 +130,8 @@ local InputDialog = InputContainer:new{
     input_type = nil,
     deny_keyboard_hiding = false, -- don't hide keyboard on tap outside
     enter_callback = nil,
+    strike_callback = nil, -- call this on every keystroke
```

Comment:
`strike_callback = nil, -- call this on every keystroke (used with TermInputText)`
(so we won't wonder why we don't find it when looking into ImputText :)